src/styles/styles/card.scss

```
@@ -0,0 +1,146 @@
+  .catalog__content {
+    box-sizing: border-box;
+    display: grid;
```

remove extra spaces in all file
```suggestion
.catalog__content {
  box-sizing: border-box; 
  display: grid;
```